shell/platform/fuchsia/dart_runner/dart_component_controller_v2.cc

```diff
@@ -213,24 +213,32 @@ bool DartComponentControllerV2::CreateAndBindNamespace() {
 
 bool DartComponentControllerV2::SetUpFromKernel() {
   dart_utils::MappedResource manifest;
+
   if (!dart_utils::MappedResource::LoadFromNamespace(
           namespace_, data_path_ + "/app.dilplist", manifest)) {
     return false;
   }
 
-  if (!dart_utils::MappedResource::LoadFromNamespace(
+  const uint8_t *isolate_data, *isolate_instructions;
+  if (dart_utils::MappedResource::LoadFromNamespace(
           nullptr, "/pkg/data/isolate_core_snapshot_data.bin",
           isolate_snapshot_data_)) {
-    return false;
+    isolate_data = isolate_snapshot_data_.address();
+  } else {
+    FX_LOG(ERROR, LOG_TAG, "Failed to load isolate snapshot data.");
+    isolate_data = nullptr;
   }
-  if (!dart_utils::MappedResource::LoadFromNamespace(
+
+  if (dart_utils::MappedResource::LoadFromNamespace(
           nullptr, "/pkg/data/isolate_core_snapshot_instructions.bin",
           isolate_snapshot_instructions_, true /* executable */)) {
-    return false;
+    isolate_instructions = isolate_snapshot_instructions_.address();
+  } else {
+    FX_LOG(ERROR, LOG_TAG, "Failed to load isolate snapshot instructions.");
```

Comment:
If we are switching snapshot_kind to 'core' instead of 'core-jit' we don't expect to find and load isolate_core_snapshot_instructions.bin, so I am not sure why this should be logged as an ERROR here, it seems just confusing to see an error being logged here.

Reply:
Yes, noted this in the PR description. What I'm trying to figure out is whether there are situations where we use the isolate snapshot instructions or not in this codepath. Could I just remove the code or should it fail silently?

Reply:
> Yes, noted this in the PR description. What I'm trying to figure out is whether there are situations where we use the isolate snapshot instructions or not in this codepath. Could I just remove the code or should it fail silently?

You would use it only in snapshot kind core-jit is used and since that is something we don't want to use I presume this code can be removed.

Reply:
Done. Removed arguments that reference the instructions snapshot and logic that loads them for core snapshots. Let me know if I went overboard with deleting things here, I'm not very familiar with the GN rules we're using here.